app/views/completed_transactions/_fields.html.erb

```diff
@@ -30,6 +30,10 @@
       <%= f.radio_button :promotion_choice, 'register_to_vote',
         { checked: (f.object.promotion_choice == "register_to_vote"), disabled: @resource.locked_for_edits? } %>
       <%= f.label :promotion_choice, "Promote register to vote", value: 'register_to_vote' %>
+      <br />
+      <%= f.radio_button :promotion_choice, 'mot_reminder',
+        { checked: (f.object.promotion_choice == "mot_reminder"), disabled: @resource.locked_for_edits? } %>
+      <%= f.label :promotion_choice, "Promote MOT Reminders", value: 'mot_reminder' %>
```

Comment:
Should this be "Promote MOT reminders", to have consistent casing with the others?

Reply:
Thanks for spotting @barrucadu !